test/test_accumulator.jl

```diff
@@ -112,33 +112,44 @@
     @test ct6["b"] == 0
     @test ct6["c"] == 4
 
-    s = ["y", "el", "sol", "se", "fue"]
-    @test counter(length(x) for x in s) == counter(map(length, s))
 
+    @testset "Generators" begin
+        s = ["y", "el", "sol", "se", "fue"]
+        @test counter(length(x) for x in s) == counter(map(length, s))
+    end
 
-    # non-integer uses
-    acc = Accumulator(Symbol, Float16)
-    acc[:a] = 1.5
-    @test acc[:a] ≈ 1.5
-    push!(acc, :a, 2.5)
-    @test acc[:a] ≈ 4.0
-    dec!(acc, :a)
-    @test acc[:a] ≈ 3.0
+    @testset "non-integer uses" begin
+        acc = Accumulator(Symbol, Float16)
```

Comment:
Not related to your changes, but let's deprecate this constructor, and add

```
Accumulator{T,V}() where {T,V<:Number} = Accumulator{T,V}(Dict{T,V}())
```

Not necessary in this PR, but it should be a three line change, so if you have time...